GOVERNANCE.md

```diff
@@ -136,12 +136,9 @@ Provide a summary of the nominee's contributions. For example:
 Mention @nodejs/collaborators in the issue to notify other Collaborators about
 the nomination.
 
-If there are no objections raised by any Collaborators one week after
-the issue is opened, the nomination will be considered as accepted.
-Should there be any objections against the nomination, the TSC is responsible
-for working with the individuals involved and finding a resolution.
-The nomination must be approved by the TSC, which is assumed when there are no
-objections from any TSC members.
+The nomination passes if no Collaborators oppose it after one week. If a
+Collaborator opposes the nomination, the TSC may still choose to approve it.
```

Comment:
Not that this has happened before, but I don't think the TSC usually insists on *approving* a nomination even though there is objection, so this somewhat changes the actual meaning of the passage?

What I have in mind when I read this is more like:

```
If a Collaborator opposes the nomination, the TSC must try to resolve the disagreement.
If the disagreement cannot be resolved within <insert time>, the nomination fails.
```

Reply:
I definitely see what you're saying, @joyeecheung. My intention was to remove the whole thing about the TSC being responsible for facilitating anything. While the TSC could *choose* to do that, I'm not sure it is terribly relevant or accurate to say that they are *responsible* for it. Moreover, such actions are far more likely to be taken by individual TSC members independently than by the TSC as a whole. Lastly, as you point out, this has literally never come up.

The whole "TSC can override the objection" thing wasn't supposed to be new material, but I see that I was reading too much into the text. So I've removed it. PTAL!